Classes/New Issue/NewIssueTableViewController.swift

```diff
@@ -141,7 +144,12 @@ final class NewIssueTableViewController: UITableViewController, UITextFieldDeleg
     /// Attempts to sends the current forms information to GitHub, on success will redirect the user to the new issue
     @objc func onSend() {
         guard let titleText = titleText else {
-            Squawk.showError(message: NSLocalizedString("You must provide a title!", comment: "Invalid title when sending new issue"))
+            Squawk.showIssueError(message: NSLocalizedString("You must provide a title!", comment: "Invalid title when sending new issue"), view: bodyField)
```

Comment:
Thoughts on rewording this alert? Maybe something kinder like:

Title: "Title Required" or "Issue requires title"
Body: "Please provide a title and try again"

Reply:
Yep, I like title option 2... and I like the body. Mind updating that, @wayni208?

Reply:
Before I resolve this and because I had made the change before you replied @BasThomas , are we good on the message as I've written it? Here are both:

```swift
        guard let titleText = titleText else {
            Squawk.showIssueError(message: NSLocalizedString("An issue title is required. Please add a title and try again.", comment: "Invalid title when sending new issue"), view: bodyField)
            return
        }
```

Also
```swift
cancelAction_onCancel(
            texts: [titleText, bodyText],
            title: NSLocalizedString("Unsaved Changes", comment: "New Issue - Cancel w/ Unsaved Changes Title"),
            message: NSLocalizedString("Are you sure you want to discard this new issue? Your message will be lost.",
                                       comment: "New Issue - Cancel w/ Unsaved Changes Message")
        )
```

Reply:
Sounds good!